iterations_runner.lua

```diff
@@ -147,7 +151,7 @@ for BM_i = 1, BM_iters, 1 do
         io.stderr:write(string.format("[iterations_runner.lua] iteration %d/%d\n", BM_i, BM_iters))
     end
     if BM_instrument then
-        jitstats.getsnapshot(stats_start)
+        jitlog.addmarker("BEGIN")
```

Comment:
Is it worth including an iteration number in your markers? Just for sanity.

Reply:
It was half place holder but adding a number to the string makes it a bit more of a hassle to parse in the reader. The flag bits are unused for the marker so i could stuff the count in there.